README.md

```diff
@@ -49,3 +49,44 @@ npm run test:integration:undeploy
 ```
 
 Performs undeploy of the app inside local OpenShift.
+
+### Helm
+
+
+To deploy the app via Helm you will need docker installed and a kubernetes cluster.
+
+First build the docker image
+
+
+```sh
+docker build -t nodejs-rest-http:1.0.0 .
```

Comment:
do we use docker in other examples already? Should use use podman or maybe show both?